test/iron-location.html

```diff
@@ -436,7 +449,6 @@
       });
 
       test('of the query portion of the URL', function() {
-        expect(window.location.search).to.be.equal('');
```

Comment:
initial query is up to the discretion of the testing framework. They may have `?dom=shadow` etc.